app/channels/application_cable/channel.rb

```diff
@@ -1,3 +1,5 @@
+# frozen_string_literal: true
+
 module ApplicationCable
   class Channel < ActionCable::Channel::Base
```

Comment:
Purpose of this class?

Reply:
seems like it is autogenerated by rails in all v5+, I believe it is used for storing shared logic between channels. As of now we don't use this, but it seems like it could be useful in the future for implementing a newsfeed